CODEOWNERS

```
@@ -0,0 +1 @@
+*   @jrevels
```

```suggestion
*   @jrevels @omus 
```

all for one and one for all